src/js/themes/base.d.ts

```diff
@@ -1151,7 +1151,16 @@ export interface ThemeType {
     };
   };
   nameValueList?: {
-    gap?: { row?: GapType; column?: GapType };
+    gap?: { 
+      pairDirectionColumn?: {
```

Comment:
This doesn't seem consistent with our pattern elsewhere of keying off of state at the root of the component, for example `theme.button.active`. Did you consider something like `theme.nameValueList.column.gap` and `theme.nameValueList.grid.gap`?

Reply:
I also like the idea of keeping the state at the root, then setting attributes specifically for each particular state.